tests/unit/utils/email-validation-test.js

```diff
@@ -0,0 +1,32 @@
+import emailValidationRegex from 'preprint-service/utils/email-validation';
+import { module, test } from 'qunit';
+
+module('Unit | Utility | email validation');
+
+test('check against allowed email domains', function(assert) {
+    const allowedDomains = [
+        'endo@domain.org',
+        'tsukahara@email.domain.edu',
+        'kasa!matsu@[123.321.456.789]',
+        'yama#waki@0ph1.1pb2.hb',
```

Comment:
Could you check with Eric to get some of the domains that have failed previously and include them in these tests? That way we're testing against the actual case we're trying to fix.